### PR TITLE
Flexible body classes

### PR DESCRIPTION
**Pull request by jasoncypret**
#### Screens
image

#### Breaking Changes

Nope

#### Runway Ticket URL

TBD

#### How to test this

Test that normal body colors are acting normal. To test the bug fix we need to add a class + a status. Dojo does  this

#### Checklist:

- [x] **DEPLOY** Please add the `Milano` label when you are ready for a review
- [x] **SCREENSHOT** Please add a screen shot or two
- [x] **SPECS** Please cover your changes with specs
- [x] **CHANGELOG** Please add an entry on `[Unreleased]` section to every functionality `ADDED`/`CHANGED`/`DEPRECATED`/`REMOVED`/`FIXED`

### CODE REVIEW

**Comment by jasoncypret** [comment]
Pushed an alpha to test. We can get this in the next release + plus the alpha if needed

**Comment by jasoncypret** [comment]
@bh247484 This was tested and looks good. I'm not sure if they need it right away...but I'll let you know shortly

File: app/pb_kits/playbook/pb_body/_body.jsx [hunk 1]
```diff
@@ -40,11 +40,11 @@ const Body = (props: BodyProps) => {
   const ariaProps = buildAriaProps(aria)
   const dataProps = buildDataProps(data)
   const classes = classnames(
-    className,
     buildCss('pb_body_kit', color, status, {
       dark: dark,
     }),
-    spacing(props)
+    spacing(props),
+    className
```
Comment (drborges) [inline app/pb_kits/playbook/pb_body/_body.jsx:47]: 👌🏼 Thanks @jasoncypret! @powerhome/thunderbolts was about to open a PR for this.

Comment (jasoncypret) [reply app/pb_kits/playbook/pb_body/_body.jsx:47]: Cool! @drborges We are merging into Nitro today. Follow this PR to see when it lands: https://github.com/powerhome/nitro-web/pull/16180 CC

**Pull request merged**